CONTRIBUTING.md

```diff
@@ -70,13 +67,13 @@ Packetbeat:
     $ cd beats/packetbeat
     $ make
 
-Some of the Beats might have extra development requirements, in which case a
-CONTRIBUTING.md file is find in the Beat directory.
+Some of the Beats might have extra development requirements, in which case you'll find a
```

Comment:
As packetbeat is the only one with a local file, we should probably remove it in the future and here link to a general developer guide which then links to packetbeat dev guide. This again points in the direction that all dev guide should leave in on place (@urso ;-) ).